src/routes/v2/authenticatedSites/index.js

```diff
@@ -225,10 +222,7 @@ const getAuthenticatedSitesSubrouter = ({
   authenticatedSitesSubrouter.use("/contactUs", contactUsV2Router.getRouter())
   authenticatedSitesSubrouter.use("/homepage", homepageV2Router.getRouter())
   authenticatedSitesSubrouter.use("/settings", settingsV2Router.getRouter())
-  authenticatedSitesSubrouter.use(
-    "/notifications",
-    notificationsRouter.getRouter()
-  )
+  authenticatedSitesSubrouter.use(notificationOnEditHandler.createNotification)
```

Comment:
double checking - we added a `next` parameter because we want control to come here and create a new notification, correct?